.github/workflows/ci.yml

```diff
@@ -181,27 +206,51 @@ jobs:
             - name: Run docker
               run: |
                   git clone https://github.com/spryker/docker-sdk.git ./docker
-                  docker/sdk boot deploy.ci.api.mariadb.yml -v
-                  docker/sdk up -t -v
+                  docker/sdk boot deploy.ci.api.mariadb.yml
+                  docker/sdk up -t
+                  docker/sdk cli composer dump-autoload -o -a --apcu
                   docker/sdk testing codecept fixtures
                   docker/sdk testing console queue:worker:start --stop-when-empty
                   docker/sdk testing codecept run -c codeception.api.yml
 
+    php-82-mariadb-acceptance-alpine-dynamic-store-off:
```

Comment:
For all newly added jobs (if they were not part of hackathon scope already, before the renaming) please add `if: "!contains(github.event.pull_request.head.ref, 'hackathon')"`.